scene/gui/color_picker.h

```diff
@@ -276,7 +276,6 @@ class ColorPicker : public VBoxContainer {
 	void _copy_color_to_hsv();
 	void _copy_hsv_to_color();
 
-	PickerShapeType _get_actual_shape() const;
```

Comment:
Looks like unrelated change.

Reply:
This change is here for nearly a year now, and https://github.com/godotengine/godot/pull/99662 has merge conflict and was opened 9 months after this PR.